### ISSUE TITLE
evaluating performance

### ISSUE THREAD

**Issue by yukimasano** [abarbu/objectnet-template-pytorch#13]
Hi,

I am trying to evaluate models on objectnet, but seem to be struggling with the correct alignment. 
I am using ImageFolder with pytorch, and a model pretrained on imagenet. This models predictions I map to Objectnet via the `imagenet_pytorch_to_objectnet_id.json` file and then compute accuracy to the class labels coming from the ImageFolder dataset. However this is leading to 0.3% performance with a pretrained imagenet model, so something must be off.

**Comment by yukimasano** [comment]
For anyone wondering, this code snippet does it: https://github.com/lucaslie/torchprune/blob/b753745b773c3ed259bf819d193ce8573d89efbb/src/torchprune/torchprune/util/datasets/objectnet.py
